pep-9999.rst

```diff
@@ -0,0 +1,1037 @@
+PEP: 9999
+Title: Graceful cooperation between external and Python package managers
+Author: Geoffrey Thomas <geofft@ldpreload.com>, Matthias Klose <doko@ubuntu.com>, Filipe Laíns <lains@riseup.net>, Donald Stufft <donald@python.org>, Tzu-Ping Chung <uranusjr@gmail.com>, Stefano Rivera <stefanor@debian.org>, Elana Hashman <ehashman@debian.org>, Pradyun Gedam <mail@pradyunsg.me>
+Discussions-To: TODO discourse
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 2021-05-18
+Post-History:
+
+Abstract
+========
+
+This PEP proposes a mechanism to indicate that a certain
+Python interpreter's package installation is managed by
+external means (such as an OS package manager) and that
+Python-specific package management tools like pip should, by
+default, neither install nor remove packages in an
+interpreter-wide context, guiding the end user towards using
+a virtual environment.
```

Comment:
This wording is really difficult to grok. I'd suggest we rewrite this paragraph. Perhaps,

> There are contexts (such as an OS package manager) where Python-specific package management tools like `pip` should neither install nor remove packages by default, and instead guide the end user towards using a virtual environment. This PEP proposes a mechanism for Python installations to communicate this.

Reply:
What do you think about replacing this paragraph with the following three?

> A long-standing practical problem for Python users has been  conflicts between OS package managers and Python-specific package management tools like pip. These conflicts include both Python-level API incompatibilities and conflicts over file ownership.
>
> Historically, Python-specific package management tools have defaulted to installing packages into an implicit global context. With the standardization and popularity of virtual environments, a better solution for most (but not all) use cases is to use Python-specific package management tools only within a virtual environment.
>
> This PEP proposes a mechanism for a Python installation to communicate to tools like pip that its global package
installation context is managed by some means external to Python, such as an OS package manager. It specifies that  Python-specific package management tools should neither install nor remove packages into the interpreter's global context, by default, and should instead guide the end user towards using a virtual environment.

Even with the last paragraph, it comes out to exactly 200 words, which is the suggestion from PEP 1 :)

Reply:
(pushed that change)